Tests/TemplatesTests/InterfaceBuilderTests.swift

```diff
@@ -12,69 +12,49 @@ class InterfaceBuilderTests: XCTestCase {
     static let all = ["empty", "all"]
   }
 
+  // swiftlint:disable:next closure_body_length
```

Comment:
Indentation of comments here and below seems strange? (Missing one space probably?)

Reply:
Erm, seems fine to me? (in Sublime, Xcode and Github)

Reply:
Hmmm strange

Here's what I see from reviewing from my phone:
![1edc9e5e-f224-453d-8117-64d1d317a15b](https://user-images.githubusercontent.com/216089/50862125-0a966d80-139b-11e9-810b-d6071bfe34a7.jpeg)

Either it's tabs vs space, or its a glitch in GitHub rendering? (But GitHub using monospaced fonts, seems strange that it would be?)

Reply:
I tried a find&replace for tabs in that file with no changes 😕 And the number of spaces before each line is the same.

Reply:
Ok… I don't see it either when I force the desktop mode on Safari mobile and see the desktop version of this GitHub page from my phone… so glitch it is I guess 🤷‍♂️